### PR TITLE
Include cdk exports

### PR DESCRIPTION
**Pull request by krishanthisera**
This PR is CDK V1 version of #290 

> Introduced a parameter call stackExportPrefix and enable export name in CloudFormation output.

### CODE REVIEW

File: packages/static-hosting/lib/static-hosting.ts [hunk 1]
```diff
@@ -53,6 +54,9 @@ export class StaticHosting extends Construct {
 
     constructor(scope: Construct, id: string, props: StaticHostingProps) {
         super(scope, id);
+        
+        // Should the stackExportPrefix is empty, 'StaticHosting' should be used as the prefix 
+        const exportPrefix =  props.exportPrefix ? props.exportPrefix :  'StaticHosting'
```
Comment (TheOrangePuff) [inline packages/static-hosting/lib/static-hosting.ts:59]: What's the advantage of having this default to `StaticHosting` rather than just a blank string?

**Review by TheOrangePuff — CHANGES_REQUESTED** [review]
Just had a question

File: packages/static-hosting/lib/static-hosting.ts (hunk 1 above)
Comment (krishanthisera) [reply packages/static-hosting/lib/static-hosting.ts:59]: Well this is act as an identifier, say if it is blank string: The bucket name would be exported as `BucketName`, which perhaps you may have already using `BucketName` as an export in one of your other stack.

Comment (TheOrangePuff) [reply packages/static-hosting/lib/static-hosting.ts:59]: Oh okay that makes sense :slightly_smiling_face:

**Review by TheOrangePuff — APPROVED** [review]

**Pull request merged**